code/modules/mob/living/carbon/alien/humanoid/life.dm

```diff
@@ -1,6 +1,6 @@
 
 
-/mob/living/carbon/alien/humanoid/proc/adjust_body_temperature(current, loc_temp, boost)
+/mob/living/carbon/alien/humanoid/proc/adjust_bodytemp(current, loc_temp, boost)
```

Comment:
Another set of pointless renames clogging the diffs.

Reply:
Who the fuck cares? Its for consistency with every other use of "bodytemp"

Reply:
if you're renaming procs at least follow basic consistency.
Old name was better in that case